### PR TITLE
[collada-dom] fix emscripten build

### PR DESCRIPTION
**Pull request by Pospelove**

### CODE REVIEW

File: ports/collada-dom/portfile.cmake [hunk 1]
```diff
@@ -9,20 +9,20 @@ vcpkg_from_github(
         use-uriparser.patch
         use-vcpkg-minizip.patch
         fix-shared-keyword.patch
+        fix-emscripten.patch
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (Cheney-W) [inline ports/collada-dom/portfile.cmake:16]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

File: ports/collada-dom/portfile.cmake [hunk 2]
```diff
@@ -9,20 +9,20 @@ vcpkg_from_github(
...
-    PREFER_NINJA
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/collada_dom-2.5)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/collada_dom-2.5)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
```
Comment (Cheney-W) [inline ports/collada-dom/portfile.cmake:23]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```

File: ports/collada-dom/portfile.cmake [hunk 3]
```diff
@@ -9,20 +9,20 @@ vcpkg_from_github(
...
 
 # Handle copyright
 file(INSTALL ${SOURCE_PATH}/licenses/license_e.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (Cheney-W) [inline ports/collada-dom/portfile.cmake:26]:
```suggestion
file(INSTALL "${SOURCE_PATH}/licenses/license_e.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

File: ports/collada-dom/vcpkg.json [hunk 4]
```diff
@@ -1,16 +1,25 @@
 {
   "name": "collada-dom",
-  "version-string": "2.5.0",
-  "port-version": 6,
+  "version-semver": "2.5.0",
```
Comment (Cheney-W) [inline ports/collada-dom/vcpkg.json:3]:
```suggestion
  "version": "2.5.0",
```

**Comment by Pospelove** [comment]
@Cheney-W thanks for your review, committed all your suggestions

**Comment by Cheney-W** [comment]
Please run below commands:
```
.\vcpkg x-add-version collada-dom --overwrite-version
git add .
git commit -m "xxx"
git push
```

**Comment by Pospelove** [comment]
@Cheney-W `nothing to commit, working tree clean`

**Comment by Pospelove** [comment]
I updated hash manually in https://github.com/microsoft/vcpkg/pull/29562/commits/d2833cb0072bb4866410d89d06c285510bacde36

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**